src/controllers/WalletController.ts

```diff
@@ -105,8 +110,8 @@ class WalletController {
     return null;
   }
 
-  on(event: EventList, callback: () => void) {
-    EventHandler.addEventHandler(event, callback);
+  on(event: any, callback: () => {}) {
```

Comment:
We should use the `EventList` string literal type here so that users of the library will get autocompletion